PatientRegistrationApp/src/components/SinglePatientFollowUp.vue

```diff
@@ -83,6 +84,17 @@ export default {
     updatePersonalInfoData(personalInfoPayload) {
       this.dataPersonalInfo = personalInfoPayload;
       this.qrValue = this.dataPersonalInfo.familyName + ", " + this.dataPersonalInfo.givenName + " " + this.dataPersonalInfo.middleName + " " + this.dataPersonalInfo.suffix
+    },
+    updateScreeningResponseData(screeningResponsesPayload) {
+      this.dataScreeningResponses = screeningResponsesPayload;
+      
+      //to do: add the screening responses to the qrValue string 
```

Comment:
I'm not sure if keeping this will conflict with MVS-634. But if it doesn't then you can keep it and I will update from development before merging MVS-634 to ensure that the code is not commented out

Reply:
Please leave a 👍 or just comment so I know this was addressed.

Reply:
Done